test/test.sh

```diff
@@ -323,6 +334,34 @@ test_server_login_retry() {
 	return $rc
 }
 
+# tcp_user_timeout
+test_tcp_user_timeout() {
+	case `uname` in
+	Linux)
+		# timeout at ~7s
+		admin "set tcp_user_timeout=2000"
```

Comment:
If I comment out this line, the test does not fail.

Reply:
😱 of course! On it.